tests/index.js

```diff
@@ -1,15 +1,47 @@
 import ENV from './-env';
 import console from '../src/shared/log';
-// import './shared-tests';
+import './shared-tests';
 import './runtime-tests';
 import './heimdall-tree-tests';
+import scenarios from './scenarios';
+
+function seekArg(name) {
+  let args = process.argv;
+
+  for (let i = 0, l = args.length; i < l; i++) {
+    if (args[i].indexOf(name) === 0) {
+      let argStr = args[i];
+
+      return {
+        index: i,
+        name: name,
+        arg: argStr,
+        value: argStr.substr(name.length + 1)
+      };
+    }
+  }
+
+  return -1;
```

Comment:
should we return `null` since the name of the function suggests that it fetches the value of the argument? abit odd to see `-1`